datatypes/papas/Helix.h

```diff
@@ -0,0 +1,47 @@
+//
+//  Helix.hpp
+//  papas
+//
+//  Created by Alice Robson on 11/05/16.
+//
+//
+
+#ifndef Helix_h
+#define Helix_h
+
+#include "Path.h"
+
+namespace papas {
+/// Decribes helix path of charged particles
+class Helix : public Path {
```

Comment:
missing doc, such as in the cluster class. 
I guess it's in the todo, so I won't comment further on documentation, except for specific points